drivers/dac/dac_mcux_lpdac.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2020 Henrik Brix Andersen <henrik@brixandersen.dk>
```

Comment:
This seems like a new driver. Do we need this Copyright Line?

Reply:
It seems like this driver is very similar to the one that brix wrote, with only a few small differences, @Albort12138 was this intentional? If so, maybe it should say  "Based on xyz_file.c, which was: " above this line

Reply:
> This seems like a new driver. Do we need this Copyright Line?

Yes, we need to keep this copyright line. Have you seen my comment above where I requested to add it because this driver is almost a complete copy of existing drivers written by Henrik?

Please see also: https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects

Adding a note "Based on xyz_file.c" is not necessary.

Reply:
Yeah, I agree with @martinjaeger.